services/QuillLMS/app/models/cron.rb

```diff
@@ -3,6 +3,8 @@
 class Cron
   # Configured in Heroku Scheduler to run every 10 minutes
   def self.interval_10_min
+    run_at_20_minute_mark if Time.now.min.between?(20, 29)
+    run_at_50_minute_mark if Time.now.min.between?(50, 59)
```

Comment:
You could take or leave this, but I'd be inclined to make sure your `now` is in UTC.  Just in case this somehow runs on servers where local time is in one of those "off by 30 minutes" timezones.

Reply:
I'd rather not have to troubleshoot some bizarre time edge case in the future, so I will peg this to UTC.

Reply:
@anathomical So, the `self.now` method uses a memoized variable which carried through with the different specs.  I've removed the timecop blocks and just used a stub instead.